### PR TITLE
Add GDCC Mainet and Testnet

### PR DESCRIPTION
**Pull request by GDCC-COIN**
Add GDCC Mainet and Testnet

### CODE REVIEW

File: _data/icons/gdcc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmdutLLSAfLaX6YehUPJcqvbf9NztgQC1gArCnYo4Ly6mu",
+    "width": 1000,
+    "height": 1628,
```
Comment (ligi) [inline _data/icons/gdcc.json:5]: identify QmdutLLSAfLaX6YehUPJcqvbf9NztgQC1gArCnYo4Ly6mu         12:42:02
QmdutLLSAfLaX6YehUPJcqvbf9NztgQC1gArCnYo4Ly6mu PNG 443x454 443x454+0+0 8-bit sRGB 19992B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-6796.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "infoURL": "https://www.gdccoin.io/",
+  "shortName": "gdcc",
+  "chainId": 6796,
+  "networkId": 6796,
+  "slip44": 714,
+  "explorers": [
+    {
+      "name": "gdcctestnet",
+      "url": "https://testnet.gdccscan.io/",
```
Comment (ligi) [inline _data/chains/eip155-6796.json:21]:
```suggestion
      "url": "https://testnet.gdccscan.io",
```

File: _data/icons/gdcc.json (hunk 1 above)
Comment (GDCC-COIN) [reply _data/icons/gdcc.json]: updated

File: _data/chains/eip155-7777a.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "infoURL": "https://www.gdccoin.io/",
+  "shortName": "gdcc",
+  "chainId": 7777,
+  "networkId": 7777,
+  "slip44": 714,
+  "explorers": [
+    {
+      "name": "gdccmainet",
+      "url": "https://gdccscan.io/",
```
Comment (ligi) [inline _data/chains/eip155-7777a.json:35]:
```suggestion
      "url": "https://gdccscan.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
EIP-7777 is already taken - read https://github.com/ethereum-lists/chains#collision-management

**Pull request closed without merging**